andela/mmdp_site
# MMDP-566 - Fix thematic pillar option display bug

**What does this PR do?**

Fixes the display of thematic pillars options on beneficiary services form and manage thematic pillar dropdown page.

**description of Task to be completed?**

Change `pillarName` to `pillarTitle`

**How should this be manually tested?**

- check out to this branch
- start development server
- navigate to `http://localhost:1234/dropdowns/thematic-pillars` in your browser
- also, navigate to beneficiary service form in the stakeholder directory.

**Any background context you want to provide?**

- None

**What are the relevant Jira issues?**

[MMDP-000](https://viisaus.atlassian.net/browse/MMDP-000)

**Screenshots (if appropriate)**

<img width="1058" alt="Screenshot 2019-05-20 at 1 24 30 PM" src="https://user-images.githubusercontent.com/22221987/58021832-fcecb380-7b03-11e9-96cf-a118c9fcda7e.png">

<img width="1125" alt="Screenshot 2019-05-21 at 4 00 54 PM" src="https://user-images.githubusercontent.com/22221987/58107410-ae154b80-7be1-11e9-9520-d5e4de0b84b0.png">


**Questions:**

- None
